### PR TITLE
Fix swagger dependecy

### PR DESCRIPTION
**Pull request by RodriSanchez1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "rand-token": "0.4.0",
     "sendgrid": "5.2.3",
     "should": "^13.2.3",
-    "swagger": "git://github.com/swagger-api/swagger-node.git#fffa399",
+    "swagger": "https://github.com/swagger-api/swagger-node.git#fffa399",
```
Comment (tomivm) [inline package.json:55]: for me "swagger" package is never used. Only uninstall the dependency

Comment (RodriSanchez1) [reply package.json]: Ok

File: package.json [hunk 2]
```diff
@@ -52,7 +52,6 @@
     "rand-token": "0.4.0",
     "sendgrid": "5.2.3",
     "should": "^13.2.3",
-    "swagger": "git://github.com/swagger-api/swagger-node.git#fffa399",
```
Comment (tomivm) [inline package.json:55]: In my opinion. Only this is required for this PR. Leave the other stuff for a different PR dedicated to removing the Swagger global dependency.

**Review by martinbedouret — APPROVED** [review]
